[INFO   ] Logger: Record log in C:\Users\DELL\.kivy\logs\kivy_21-12-09_2.txt
[INFO   ] deps: Successfully imported "kivy_deps.angle" 0.3.0
[INFO   ] deps: Successfully imported "kivy_deps.glew" 0.3.0
[INFO   ] deps: Successfully imported "kivy_deps.sdl2" 0.3.1
[INFO   ] Kivy: v2.0.0
[INFO   ] Kivy: Installed at "C:\Users\DELL\anaconda3\lib\site-packages\kivy\__init__.py"
[INFO   ] Python: v3.8.8 (default, Apr 13 2021, 15:08:03) [MSC v.1916 64 bit (AMD64)]
[INFO   ] Python: Interpreter at "C:\Users\DELL\anaconda3\python.exe"
[INFO   ] Image: Providers: img_tex, img_dds, img_sdl2, img_pil (img_ffpyplayer ignored)
[INFO   ] Factory: 186 symbols loaded
[INFO   ] Text: Provider: sdl2
[INFO   ] Audio: Providers: audio_sdl2 (audio_ffpyplayer ignored)
[INFO   ] Window: Provider: sdl2
[INFO   ] Window: Activate GLES2/ANGLE context
[INFO   ] GL: Using the "OpenGL" graphics system
[INFO   ] GL: Backend used <angle_sdl2>
[INFO   ] GL: OpenGL version <b'OpenGL ES 3.0.0 (ANGLE 2.1.13739 git hash: 385fb40fd460)'>
[INFO   ] GL: OpenGL vendor <b'Google Inc.'>
[INFO   ] GL: OpenGL renderer <b'ANGLE (Microsoft Basic Render Driver Direct3D11 vs_5_0 ps_5_0)'>
[INFO   ] GL: OpenGL parsed version: 3, 0
[INFO   ] GL: Shading version <b'OpenGL ES GLSL ES 3.00 (ANGLE 2.1.13739 git hash: 385fb40fd460)'>
[INFO   ] GL: Texture max size <16384>
[INFO   ] GL: Texture max units <16>
[INFO   ] Window: auto add sdl2 input provider
[INFO   ] Window: virtual keyboard not allowed, single mode, not docked
[INFO   ] Base: Start application main loop
[INFO   ] GL: NPOT texture support is available
[INFO   ] Base: Leaving application in progress...
[WARNING] stderr: Traceback (most recent call last):
[WARNING] stderr:   File "C:\Users\DELL\anaconda3\lib\runpy.py", line 194, in _run_module_as_main
[WARNING] stderr:     return _run_code(code, main_globals, None,
[WARNING] stderr:   File "C:\Users\DELL\anaconda3\lib\runpy.py", line 87, in _run_code
[WARNING] stderr:     exec(code, run_globals)
[WARNING] stderr:   File "invaders\__main__.py", line 28, in <module>
[WARNING] stderr:     Invaders(width=GAME_WIDTH,height=GAME_HEIGHT).run()
[WARNING] stderr:   File "invaders\game2d\app.py", line 279, in run
[WARNING] stderr:     kivy.app.App.run(self)
[WARNING] stderr:   File "C:\Users\DELL\anaconda3\lib\site-packages\kivy\app.py", line 950, in run
[WARNING] stderr:     runTouchApp()
[WARNING] stderr:   File "C:\Users\DELL\anaconda3\lib\site-packages\kivy\base.py", line 582, in runTouchApp
[WARNING] stderr:     EventLoop.mainloop()
[WARNING] stderr:   File "C:\Users\DELL\anaconda3\lib\site-packages\kivy\base.py", line 347, in mainloop
[WARNING] stderr:     self.idle()
[WARNING] stderr:   File "C:\Users\DELL\anaconda3\lib\site-packages\kivy\base.py", line 387, in idle
[WARNING] stderr:     Clock.tick()
[WARNING] stderr:   File "C:\Users\DELL\anaconda3\lib\site-packages\kivy\clock.py", line 733, in tick
[WARNING] stderr:     self.post_idle(ts, self.idle())
[WARNING] stderr:   File "C:\Users\DELL\anaconda3\lib\site-packages\kivy\clock.py", line 776, in post_idle
[WARNING] stderr:     self._process_events()
[WARNING] stderr:   File "kivy\_clock.pyx", line 616, in kivy._clock.CyClockBase._process_events
[WARNING] stderr:   File "kivy\_clock.pyx", line 649, in kivy._clock.CyClockBase._process_events
[WARNING] stderr:   File "kivy\_clock.pyx", line 645, in kivy._clock.CyClockBase._process_events
[WARNING] stderr:   File "kivy\_clock.pyx", line 218, in kivy._clock.ClockEvent.tick
[WARNING] stderr:   File "invaders\game2d\app.py", line 356, in _refresh
[WARNING] stderr:     self.update(dt)
[WARNING] stderr:   File "invaders\app.py", line 162, in update
[WARNING] stderr:     self._wave = Wave()
[WARNING] stderr:   File "invaders\wave.py", line 137, in __init__
[WARNING] stderr:     pewSound = Sound('pewl.wav')
[WARNING] stderr:   File "invaders\game2d\sound.py", line 79, in __init__
[WARNING] stderr:     assert GameApp.is_sound(source), 'source %s is not a sound file' % repr(source)
[WARNING] stderr: AssertionError: source 'pewl.wav' is not a sound file
